.github/workflows/algolia-upload-index.yml

```diff
@@ -13,8 +13,8 @@ jobs:
     runs-on: ubuntu-20.04
 
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
```

Comment:
New major versions of these actions run on Node 16 instead of Node 12.
- https://github.com/actions/checkout/releases
- https://github.com/actions/setup-node/releases